docs/source/hardware/low_priority_faqs.md

```diff
@@ -0,0 +1,210 @@
+::::
+
+::::{dropdown} 6. How do I submit a job to the `lowpriority` partition?
+The `lowpriority` partition is a distinct partition that must be requested explicitly. SLURM accepts a comma-separated list of partitions. If a job is scheduled to run in the `lowpriority` partition, it may be preempted and re-queued. This does not happen in other partitions. The `lowpriority` partition is the only partition that can be preempted. See examples in using {ref}`low-priority`.
+::::
+
+::::{dropdown} 7. How can I ensure that my `lowpriority` partition job can run on nodes with sufficient resources?
+It is important to specify the required resources in the job script, including memory, CPUs and GPUs. For CPUs, this can include the number of cores and the micro-architecture and for GPUs this is typically handled by “class”. The GPU classes that are configured on the HPC system include P100, V100, T4, and A100. See our documentation on {ref}`slurm-guide-index` and {ref}`working-gpus` for additional information.
+::::
+
+::::{dropdown} 8. What is the downside of submitting a job to the `lowpriority` partition?
+If a job is submitted to the `lowpriority` partition and a high priority job comes through that requires resources currently occupied by the low priority job, then that low priority job will be stopped/suspended within 30s and re-queued. 
+::::
+
+::::{dropdown} 9. When should I NOT use the `lowpriority` partition to run my jobs?
+Jobs running on the `lowpriority` partition always carry the risk of being suspended before their wall time ends if a high priority job requests those resources while the low priority job is running. If you have checkpointing (see {ref}`checkpoint-jobs`) implemented in your workflow, such abrupt suspension of jobs would not be an issue, since your intermediate calculations/data are saved, and you can re-start your jobs from the point of their suspension. However, if you do not have checkpointing techniques implemented, and/or you expect your jobs to run for a while (and re-running them in the event of preemption means it is going to start all over again and take even longer to complete), it is not ideal to use the `lowpriority` partition. 
+::::
+
+:::{button-link} faq.html
+:color: primary
+:align: center
+Back to the top
+:::
+## Resources
+
+::::{dropdown} 1. What are the time limits on the `lowpriority` partition?
+Partition limits for the `lowpriority` partition are the same as the default partition (`short`), which has a 4h default walltime and a 24h max-walltime.
+::::
+
+::::{dropdown} 2. What are the resource and job limits on the `lowpriority` partition?
+Currently, the resource limits and the number for CPU-supported low priority jobs on the `lowpriority` partition are the same as the default partition (see {ref}`partition-names`). The resource limits and the number for GPU-supported low priority jobs on the `lowpriority` partition are the same as the gpu partition (see {ref}`partition-names`).
```

Comment:
```suggestion
Currently, the resource limits and the number for CPU-supported low priority jobs on the `lowpriority` partition are the same as the default partition (see {ref}`partition-names`). The resource limits and the number for GPU-supported low priority jobs on the `lowpriority` partition are the same as the `gpu` partition (see {ref}`partition-names`).

You can find more information about all available partitions here. 
```

Please remove the two "see partitions' links and just make the last sentence I added a link to that page.